doc/mgmt/SONiC_NAT_MgmtSupport_HLD.md

```diff
@@ -0,0 +1,582 @@
+             |     +--rw ietf-nat-ext:twice_nat_id?   uint16
+             |     +--ro ietf-nat-ext:statistics
+             |        +--ro ietf-nat-ext:dnat-translations-pkts?    yang:zero-based-counter64
+             |        +--ro ietf-nat-ext:dnat-translations-bytes?   yang:zero-based-counter64
+             |        +--ro ietf-nat-ext:snat-translations-pkts?    yang:zero-based-counter64
+             |        +--ro ietf-nat-ext:snat-translations-bytes?   yang:zero-based-counter64
+             +--rw ietf-nat-ext:nat_timeout?       uint32
+             +--rw ietf-nat-ext:nat_tcp_timeout?   uint32
+             +--rw ietf-nat-ext:nat_udp_timeout?   uint16
+             +--rw ietf-nat-ext:zone-counters
+             |  +--rw ietf-nat-ext:zone-counter-entry* [zone-id]
+             |     +--rw ietf-nat-ext:zone-id                        uint8
+             |     +--rw ietf-nat-ext:nat-dnat-discards?             yang:zero-based-counter64
+             |     +--rw ietf-nat-ext:nat-snat-discards?             yang:zero-based-counter64
+             |     +--rw ietf-nat-ext:nat-dnat-translation-needed?   yang:zero-based-counter64
+             |     +--rw ietf-nat-ext:nat-snat-translation-needed?   yang:zero-based-counter64
+             |     +--rw ietf-nat-ext:nat-dnat-translations?         yang:zero-based-counter64
+             |     +--rw ietf-nat-ext:nat-snat-translations?         yang:zero-based-counter64
+             +--rw ietf-nat-ext:nat-pool
+             |  +--rw ietf-nat-ext:nat-pool-entry* [pool-name]
+             |     +--rw ietf-nat-ext:pool-name           string
+             |     +--rw (ietf-nat-ext:nat-ip)?
+             |     |  +--:(ietf-nat-ext:ip-address)
+             |     |  |  +--rw ietf-nat-ext:IP-ADDRESS?         inet:ipv4-address
+             |     |  +--:(ietf-nat-ext:ip-address-range)
+             |     |     +--rw ietf-nat-ext:IP-ADDRESS-RANGE?   string
+             |     +--rw ietf-nat-ext:nat-port?           string
+             +--rw ietf-nat-ext:nat-bindings
+                +--rw ietf-nat-ext:nat-binding-entry* [binding-name]
+                   +--rw ietf-nat-ext:binding-name    string
+                   +--rw ietf-nat-ext:nat-pool        string
+                   +--rw ietf-nat-ext:access-list?    string
+                   +--rw ietf-nat-ext:nat-type?       enumeration
+                   +--rw ietf-nat-ext:twice-nat-id?   uint16
+
+```
+
+SONiC NAT Yang will be used for Config Validation purposes.
+
+https://github.com/project-arlo/sonic-mgmt-framework/blob/nat-impl/models/yang/sonic/sonic-nat.yang
+
+### 3.6.2 CLI
+SONiC NAT Click CLI based Configuration and Show Commands will be supported by management framework.
+Refer Section 3.8 from https://github.com/Azure/SONiC/blob/dc5d3a894618bcb07a3c5d2dd488caf3beb7479a/doc/nat/nat_design_spec.md
+
+#### 3.6.2.1 Configuration Commands
+
+###### Add static NAT entry
+
+`static basic <global-ip> <local-ip> {snat | dnat} { twice_nat_id <value> }`
```

Comment:
{ snat | dnat } is a also a optional parameter.

Reply:
Addressed